code/src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Moment from 'moment';
+
+import { Title } from 'styling/HeaderStyling';
+
+const Header = () => {
+
+    return <Title>{Moment().format("MMM Do YY")}</Title>   
```

Comment:
Its really cool that you used moment to show the current date. I would reccomend you use the same to show the time or duration of the time since the Todo was added.